presto-jdbc/pom.xml

```diff
@@ -142,7 +142,13 @@
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>concurrent</artifactId>
-            <scope>test</scope>
+            <exclusions>
```

Comment:
Ok. I found the issue. `presto-jdbc` and `presto-main` modules are used in `presto-product-tests` module. If you didn't exclude `jmxutils` from the `presto-jdbc` the `presto-jdbc` artifact will have all the classes packed in it's fat-jar. `presto-main` relies on `jmxutils` transitively. So, the `presto-product-tests` has two different jars with the same classes. `jmxutils` itself, and the `presto-jdbc` with all the `jmxutils` classes packed. That's why it fails. 

If you don't want to exclude `jmxutils` library, you can rellocate all the classes from `org.weakref.jmx` package, similar to how it is done for other libraries (check the `maven-shade-plugin` settings in `presto-jdbc` module)